hedera-mirror-common/src/main/java/com/hedera/mirror/common/domain/transaction/RecordFile.java

```diff
@@ -72,6 +73,23 @@ public class RecordFile implements StreamFile<RecordItem> {
     @ToString.Exclude
     private String fileHash;
 
+    private Long gasUsed;
+
+    /**
+     * This field was created to be used during the sum of the total gas used, to avoid boxing/unboxing of
+     * {@link RecordFile#gasUsed}, since {@link RecordFile#gasUsed} must be a {@link Long} to ensure we can load
+     * a {@link RecordFile} from the database with {@code gasUsed = null}.
+     */
+    @EqualsAndHashCode.Exclude
+    @ToString.Exclude
+    @Transient
+    private long gasUsedAccumulator = 0;
+
+    @EqualsAndHashCode.Exclude
+    @ToString.Exclude
+    @Transient
+    private final LogsBloomFilter logsBloomFilter = new LogsBloomFilter();
```

Comment:
Should add `transient` to the fields as well since `@Transient` is specific to JPA and is not taken into account for serialization or JSON.
```suggestion
    private transient final LogsBloomFilter logsBloomFilter = new LogsBloomFilter();
```

Reply:
Done.